Bottom: ca2388b9fec3713cd1c0ee00adb0ce3ac2baf5a0
Top:    c123432bab33d032db1bc4d3a8ce9a11ee467eca
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-26 20:14:40 +0100

draw.c: Remove obsolete comment


---

diff --git a/src/draw.c b/src/draw.c
index 53757f4..20a269d 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -108,17 +108,13 @@ AddPart (void *b)
 }
 
 /*
- * initiate the actual drawing to the pixmap/screen
- * make the update block slightly larger to handle round-off
- * caused by the TO_SCREEN operation
+ * initiate the actual redrawing of the updated area
  */
 void
 Draw (void)
 {
   HideCrosshair ();
 
-  /* clear and create event if not drawing to a pixmap
-   */
   if (Block.X1 <= Block.X2 && Block.Y1 <= Block.Y2)
     gui->invalidate_lr (Block.X1, Block.X2, Block.Y1, Block.Y2);
